### ISSUE TITLE
Error adding DNS record for multi-part TLDs

### ISSUE THREAD

**Issue by marlonlamancio** [kovshenin/sail#2]
Hi, I'm sorry to bother you again but I'm having another problem. I can't add my domain .com.br (Brazil ccTLD), what could be causing this error?

```
sail domain add growthdesigner.com.br

# com.br
- Error adding DNS record for growthdesigner.com.br: Add parent domain first
```

**Comment by marlonlamancio** [comment]
I managed to solve through the `sail domain make-primary` command:

```
sail domain make-primary growthdesigner.com.br
# Updating primary domain
- Scheduled make-primary for growthdesigner.com.br
- Waiting for make-primary to complete
- Current primary domain: 20749d7700a0bb2a.sailed.io
- Running wp search-replace
- Flushing object cache
- Renaming droplet
- Primary domain updated successfully
```

**Comment by marlonlamancio** [comment]
Actually, it didn't work. I had to access the DO panel and add the domain manually, I changed the NS of my domain pointing to  DO as on doc, I checked the DNS records everything is ok, but it's returning error 404 and I couldn't identify the reason.

**Comment by kovshenin** [comment]
Hmm, seems like it's treating `com.br` as a regular domain and not a TLD. I'll look into this and let you know when it's fixed. In the meantime I think you can manually add the necessary A records at DO to point to your droplet, then use `make-primary` and `make-https` with Sail, which I'm guessing you already did.

With regards to the 404, looks like you perhaps deleted the application files from the server somehow? Maybe you deleted them locally and deployed? Can you try doing a rollback to a previous release or the first release (it's called 1337 if it's still there). You can get a list of available releases with `sail rollback` and then `sail rollback <number>`.

Let me know how it goes.

**Comment by kovshenin** [comment]
Oh actually it seems like the application files are in place, because I can sort-of access the site using the internal 20749d7700a0bb2a.sailed.io domain, which is issuing the redirect since you set your custom domain as primary. So no need to roll back, never mind on that.

I'll fix the .com.br TLD issue, then you should be able to add it as a domain as well as get an SSL cert for it.

Thanks for your patience!

**Comment by kovshenin** [comment]
@marlonlamancio I've added .com.br to the supported list of multi-part TLDs, you should be able to add the domain now, make it primary, and request an SSL cert. If your domain is pointed at DO's name servers, it should all work immediately.

With regards to the 404, I think it's a different bug which I'm looking into.

Thanks for trying out Sail!

**Comment by soulseekah** [comment]
Maybe all these should be supported https://tld-list.com/tlds-from-a-z

**Comment by kovshenin** [comment]
This should be fixed by #14 with `tldextract`
